scripts/_lines_of_code.py

```diff
@@ -65,11 +76,20 @@ def __init__(self, owned_repository: Repository, user):
         self.total_lines_of_addition = 0
         self.total_lines_of_deletion = 0
         self.name = owned_repository.name
-        for contribution in owned_repository.get_stats_contributors():
-            if contribution.author.id == user.id:
-                for week in contribution.weeks:
-                    self.total_lines_of_addition += week.a
-                    self.total_lines_of_deletion += week.d
+
+        if (owned_repository.get_commits().totalCount >= 100):
+            execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')
+
+        for commit in owned_repository.get_commits():
+            if commit.author.id == user.id:
+                for file in commit.files:
+                    if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
+                        self.total_lines_of_addition += file.additions
+                        self.total_lines_of_deletion += file.deletions
```

Comment:
```commit.author``` is defined as None when it is not commited by a Github user (likely commited via workflow). we can just ignore them (by using ```if not commit.author is None```).
```suggestion
        for commit in owned_repository.get_commits():
        if commit.author is not None:
                if commit.author.id == user.id:
                    for file in commit.files:
                        if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
                            self.total_lines_of_addition += file.additions
                            self.total_lines_of_deletion += file.deletions
```